src/humanPose/draw.js

```diff
@@ -1256,6 +1258,10 @@ function updateJointsAndBones(poseRenderInstance, poseObject, timestamp) {
 
     const poseHasParent = poseObject.parent && (poseObject.parent !== 'none');
     const pose = new Pose(jointPositions, jointConfidences, timestamp, {poseObjectId: poseObject.uuid, poseHasParent: poseHasParent});
+    pose.metadata.previousPose = mostRecentPoseByObjectId[poseObject.uuid];
```

Comment:
Just a side question: Where HumanPoseObject or generic ObjectModel get uuid property? In server code, they are defined just with objectId.

Reply:
I think this is a holdover from old code, I'll replace it with objectId everywhere in my next commit